lib/widgets/meal_item.dart

```diff
@@ -121,20 +121,20 @@ class MealItem extends StatelessWidget {
                         Icons.work,
                       ),
                       SizedBox(
-                        width: 6,
+                        width: 6
                       ),
                       Text(complexityText),
                     ],
                   ),
                   Row(
                     children: [
                       Icon(
-                        Icons.attach_money,
+                        Icons.attach_money
                       ),
                       SizedBox(
-                        width: 6,
+                        width: 6
                       ),
-                      Text(affordabilityText),
+                      Text(affordabilityText)
```

Comment:
you must have misunderstood... this is the example for not having trailing commas inside the `Text` widget's property. do not remove the trailing comma since this is one of `Row`'s children widget.